frontend-opal/src/thunks/searchResult.js

```diff
@@ -1,33 +1,113 @@
 import actions from 'actions/searchResult';
 
-const data = [
-  {
-    name: 'MS',
-    fullName: 'Morgan Stanley',
-    market: 'NYSE'
-  },
-  {
-    name: 'MSFT',
-    fullName: 'Microsoft Corp.',
-    market: 'NASDAQ'
-  },
-  {
-    name: 'FBMS',
-    fullName: 'First Branchshares Inc/MS',
-    market: 'NASDAQ'
+import { IEXCLOUD_PUBLIC_KEY } from 'react-native-dotenv';
+
+import symbolData from './symbolData';
+
+// const data = [
+//   {
+//     name: 'MS',
+//     fullName: 'Morgan Stanley',
+//     market: 'NYSE'
+//   },
+//   {
+//     name: 'MSFT',
+//     fullName: 'Microsoft Corp.',
+//     market: 'NASDAQ'
+//   },
+//   {
+//     name: 'FBMS',
+//     fullName: 'First Branchshares Inc/MS',
+//     market: 'NASDAQ'
+//   }
+// ];
+
+async function fetchData() {
+  const symbols = `https://cloud.iexapis.com/stable/ref-data/symbols?token=${IEXCLOUD_PUBLIC_KEY}`;
```

Comment:
@Yuan-EPAM extract this into a `stockApi` file and separate the `baseUrl` as a constant